src/mapclient/view/managers/options/qt/optionsdialog.ui

```diff
@@ -85,6 +85,35 @@
          </layout>
         </widget>
        </item>
+       <item>
+        <widget class="QGroupBox" name="groupBox_2">
+         <property name="title">
+          <string>MessageBox display timer</string>
+         </property>
+         <layout class="QHBoxLayout" name="horizontalLayout_2">
+          <item>
+           <widget class="QDoubleSpinBox" name="doubleSpinBoxMessageBoxTimer">
+            <property name="maximumSize">
+             <size>
+              <width>120</width>
+              <height>16777215</height>
+             </size>
+            </property>
+            <property name="value">
+             <double>2.000000000000000</double>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QLabel" name="label">
+            <property name="text">
+             <string>seconds</string>
```

Comment:
Auto close after (seconds)

Reply:
Also need to say that setting a value of zero will disable the display of the notification.

Reply:
We don't want negative numbers.

Reply:
We want to display 3 decimal places.

Reply:
I've made an additional commit.

The label has been changed to say "Auto close after (seconds)", and I've added a ToolTip to the SpinBox which explains that setting a value of zero will disable the display of the notification.

I have changed the number of decimal points displayed from 2 to 3.

As far as I can tell, the SpinBox already restricts negative numbers. Let me know if I'm mistaken.